### PR TITLE
Fix Escape Key Bug

### PR DESCRIPTION
**Pull request by apeterson7**
This PR adds a stateReducer function to the Downshift element that captures the escape key event and overrides the default behavior and closes the menu.  The default behavior is to pass null the onChange callback, which was causing the error.

### CODE REVIEW

File: src/components/MapSelectors/GlobalSelector.js [hunk 1]
```diff
@@ -235,10 +235,24 @@ const GlobalSelector = ({ handleChange, extendRight }) => {
     }
   };
 
+  const escapeKeyStateReducer = (state, changes) => {
```
Comment (mikelehen) [inline src/components/MapSelectors/GlobalSelector.js:238]: Can you add a comment explaining what's going on here?

File: src/components/MapSelectors/GlobalSelector.js [hunk 2]
```diff
@@ -235,10 +235,24 @@ const GlobalSelector = ({ handleChange, extendRight }) => {
...
+    console.log(changes);
```
Comment (mikelehen) [inline src/components/MapSelectors/GlobalSelector.js:239]: Perhaps remove this?  Or else add some surrounding text so somebody seeing it in the console knows what it is.

Comment (apeterson7) [reply src/components/MapSelectors/GlobalSelector.js]: I meant to remove this, thanks!

File: src/components/MapSelectors/GlobalSelector.js (hunk 1 above)
Comment (apeterson7) [reply src/components/MapSelectors/GlobalSelector.js:242]: Will do!

**Comment by mikelehen** [comment]
Thanks for this!

**Pull request merged**